libs/shared/src/features/audio-playback/audio-playback.component.scss

```diff
@@ -0,0 +1,12 @@
+.audio-button{
+    outline: none;
+    background-color: inherit;
+    border: none;
+    padding: 5px;
+    .mat-icon{
+        color: #7292C7;
```

Comment:
nit(non-blocking): might be good to link the color to the primary/secondary pallette. You could use `var(--color-secondary)` to get the deeper blue, or if you want to register a lighter variant you can define it in `libs\theme\src\themes\_mixins.scss`, testing with something like
```
  --color-secondary-200: #{map-get($accent, 200)};
```
The full list of colors that could be registered can be seen in `libs\theme\src\themes\picsa-default.scss` (currently just the 500 and 50 weight variants are in use)